src/components/IconTooltip/index.tsx

```diff
@@ -9,7 +9,6 @@ import { useDeviceInfo } from '../../utils';
 const useStyles = makeStyles((theme: Theme) => ({
   arrow: {
     color: theme.palette.ClrBaseGray800,
-    marginLeft: '3px',
```

Comment:
I removed this because the tooltip arrow was being decentered by it:
![image](https://user-images.githubusercontent.com/114949086/195647371-54f72048-850c-4413-b5f4-b7ddbb852ee1.png)

Reply:
@joshbuchea if you would weigh in on this change please

Reply:
I had added the margin for the previous version of the tooltip. Change looks good (as tested in storybook).

Reply:
we may need this in Josh's PR maybe?

Reply:
I [made the request](https://github.com/terraware/web-components/pull/79#issuecomment-1277868035) there 🙂